Loop/Views/BolusEntryView.swift

```diff
@@ -42,10 +42,9 @@ struct BolusEntryView: View, HorizontalSizeClassOverride {
                 .padding(.top, self.shouldAutoScroll(basedOn: geometry) ? -200 : -28)
                 .listStyle(GroupedListStyle())
                 .environment(\.horizontalSizeClass, self.horizontalOverride)
-                
                 self.actionArea
-                    .frame(height: self.isKeyboardVisible ? 0 : nil)
-                    .opacity(self.isKeyboardVisible ? 0 : 1)
+                    .frame(height: self.viewModel.isManualGlucoseEntryEnabled || self.isKeyboardVisible ? 0 : nil)
```

Comment:
Hm...this looks suspicious to me.   What does this change achieve?

Reply:
Oh... good catch. I was exploring alternatives with the animation and forgot to remove this.